CHANGELOG.md

```diff
@@ -65,7 +65,7 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - Custom log formatting (along with `colored` and `term` dependencies) has been
   removed in favor of `env_logger`. (#857).
 
-- Several dependencies have been updated. (#861)
+- Several dependencies have been updated. (#861, #865)
```

Comment:
@ilammy 
`actix` and `actix-web` dependencies have beet updated to 0.7.3 version (#861, #865) ?

Reply:
Yes, but I think this is a 'minor' update in the sense that it does not break Exonum's API, so it does not warrant any specific mention in the changelog. The link to the pull request should be enough.